### DISCUSSION TITLE
Rivet HTTP Error Pages

### DISCUSSION THREAD

**Discussion by scottanthonymurray** [indiana-university/rivet-source#419] [Feature and enhancement requests]
It would be useful to have Rivet designs for common HTTP errors (i.e. 400, 401, 403, 404, 500). Including a small sample below that are regarded to be good examples by some online sources.

![image](https://user-images.githubusercontent.com/40362172/51328841-3d4b0080-1a42-11e9-9f4c-ed0b0488d4bc.png)

![image](https://user-images.githubusercontent.com/40362172/51328866-4a67ef80-1a42-11e9-9262-03d63d9e598a.png)

![image](https://user-images.githubusercontent.com/40362172/51328896-55bb1b00-1a42-11e9-9539-782b09cb7950.png)

![image](https://user-images.githubusercontent.com/40362172/51328923-623f7380-1a42-11e9-885a-7b212f36b363.png)

**Comment by scottanthonymurray** [comment]
@levimcg Is this something we might want to include in the Rivet beta?

**Reply by levimcg** [reply]
Yes, absolutely! This has been a long-standing request. We should definitely include something like _Error page_ as a Layout in the Beta.
